witchcraft-server/src/logging/service.rs

```diff
@@ -272,47 +139,3 @@ fn log_panics() {
         }
     }));
 }
-
-#[cfg(test)]
-mod test {
-    use super::*;
-    use witchcraft_log::Level;
-
-    #[test]
-    fn loggers() {
```

Comment:
possibly worth leaving a test along these lines in place to ensure the integration with witchcraft-rust works as expected. I don't think this test itself tests much that isn't handled by the logging crate outside of the `LoggingConfig`

Reply:
Oh yeah that's fair. I'll add it back.